samples/BikeSharingApp/Bikes/azds_pipeline.yaml

```diff
@@ -156,23 +150,23 @@ stages:
             inputs:
               kubernetesServiceEndpoint: $(k8sServiceConnection)
               command: create
-              arguments: 'ns $(system.pullRequest.sourceBranch)'
+              arguments: 'ns $(k8sNamespaceforpr)'
               checkLatest: true
 
           - task: Kubernetes@1
             displayName: 'Add devspace label'
             inputs:
               kubernetesServiceEndpoint: $(k8sServiceConnection)
               command: label
-              arguments: '--overwrite ns $(system.pullRequest.sourceBranch) azds.io/space=true'
+              arguments: '--overwrite ns $(k8sNamespaceforpr) azds.io/space=true'
               checkLatest: true
 
           - task: Kubernetes@1
             displayName: 'Setup root devspace to dev'
             inputs:
               kubernetesServiceEndpoint: $(k8sServiceConnection)
               command: label
-              arguments: '--overwrite ns $(system.pullRequest.sourceBranch) azds.io/parent-space=$(k8sNamespace)'
+              arguments: '--overwrite ns $(k8sNamespaceforpr) azds.io/parent-space=$(k8sNamespace)'
```

Comment:
Can we use CamelCase as a naming convention for naming variable the variable: k8sNamespaceforpr